internal/pkg/describe/backend_service_test.go

```diff
@@ -112,8 +119,35 @@ func TestBackendServiceDescriber_Describe(t *testing.T) {
 			},
 			wantedError: fmt.Errorf("retrieve platform: some error"),
 		},
+		"return error if fail to retrieve rollback alarm names": {
```

Comment:
Would we want one more unit test that tests "return error if fail to retrieve deployment type" ?